### DISCUSSION TITLE
Reverse CompositeSprite?

### DISCUSSION THREAD

**Discussion by choephix** [jnsmalm/pixi3d#183] [Q&A] [answered]
I see in the docs, that we have a `CompositeSprite` class for rendering 3D content onto a regular 2D pixi sprite.

Is there a way to achieve the opposite? Like, if I have a regular PixiJs Container with relatively complex contents, just slap that on a Pixi3D container, maybe as a flat single mesh, and avoid worrying about z-managing that 2D container's many overlapping children?

Also since it doesn't have any, I guess assume it's `z` to be `0`, then we just manipulate the parent Pixi3D container for positioning in 3D space.

**Comment by jnsmalm** [comment, marked as answer]
It's not supported at this point. You could however render your container to a texture and then have that texture on a Sprite3D.
